README.md

```diff
@@ -125,10 +125,10 @@ res1: Double = 5.666666666666667
 
 ## Documentation
 
-Documentation is available at https://geotrellis.io/documentation.
+Documentation is available at [geotrellis.io/documentation](https://geotrellis.io/documentation).
 
 _Scaladocs_ for the the `master` branch are
-[available here](http://geotrellis.github.com/scaladocs/latest/#geotrellis.package).
+[available here](https://geotrellis.github.io/scaladocs/2.0/#geotrellis.package).
```

Comment:
How about https://geotrellis.github.io/scaladocs/latest/geotrellis/index.html?search=geotrellis?
2.0. is way too outdated and unsupported. (That is the scaladocs repo https://github.com/geotrellis/scaladocs)